docs/reference/vppcontainers/twocontainers.rst

```diff
@@ -0,0 +1,117 @@
+.. _twocontainers:
+
+.. toctree::
+
+
+=====================================
+VPP with two Containers using Vagrant
+=====================================
+
+
+Overview
+________
+
+This section will cover how to install a VM (Virtual Machine) for Vagrant, and how to use containers inside that VM.
+
+Containers are environments similar to VM's, but are known to be faster since they do not simulate  seperate kernels and hardware, as VM's do. You can read more about `Linux containers here <https://linuxcontainers.org/>`_.
+
+
+In this section, we'll use Vagrant to run our VirtualBox VM. **Vagrant** automates the configuration of virtual environments by giving you the ability to create and destroy VM's quick and seemlessly.
+
+Installing VirtualBox
+_____________________
+
+First, download VirtualBox, which is virtualization software for creating VM's.
+
+If you're on CentOS, follow the `steps here <https://wiki.centos.org/HowTos/Virtualization/VirtualBox#head-81de410879b8e7f18a127f638160e036ab99684e>`_.
+
+
+If you're on Ubuntu, perform:
+
+.. code-block:: shell
+
+   $ sudo apt-get install virtualbox 
+
+
+
+Installing Vagrant
+__________________
+
+
+Now its time to install Vagrant.
+
+Here we are on a 64-bit version of CentOS, downloading and installing Vagrant 2.1.1:
+
+.. code-block:: shell
```

Comment:
Can we refer people to the hashicorp downloads page.

https://www.vagrantup.com/downloads.html

Referencing a specific version is going to get out of date very fast here.